package.json

```diff
@@ -29,6 +31,11 @@
     "rxjs": "6.4.0"
   },
   "devDependencies": {
+    "@bazel/bazel": "latest",
+    "@bazel/buildifier": "latest",
+    "@bazel/hide-bazel-files": "0.32.2",
```

Comment:
Could be removed if `always_hide_bazel_files = True,` is used in `yarn_install`.
```suggestion
```